README.md

```diff
@@ -27,33 +27,44 @@ GRChombo is licensed under the BSD 3-Clause License. Please see LICENSE for deta
 
 ## Fawcett Tutorial
 
-First we need to get a copy of both the Chombo and GRChombo repositories.
+The following tutorial has been created to be aimed at Masters/early PhD
+students with some HPC experience. We also assume that you have an account set
+up on fawcett, and that you are happy with using GitHub. Inevitably it will not be 
```

Comment:
I thought we might as well add a link to the fawcett documentation up here (even though you did it below as well) and we should say "familarity with git" rather than "GitHub".
```suggestion
up on [fawcett](https://www.maths.cam.ac.uk/computing/faculty-hpc-system-fawcett), and that you are happy with using `git`. Inevitably it will not be 
```